ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSwindow.cpp

```diff
@@ -33,6 +33,10 @@
 #define byte __windows_byte_workaround
 #include <windows.h>
 #undef byte
+// mingw32 cross-compile bug workaround below
+#ifndef MAPVK_VK_TO_VSC_EX
+#define MAPVK_VK_TO_VSC_EX 0x04
+#endif
```

Comment:
Apparently it's Windows Vista and up. Without it I don't know how else to get whether a virtual key mapped to a scan code is an extended scan code or not. Perhaps it wasn't possible to determine that before Windows Vista.

It's not a big deal, I can remove this if we want, as the keyboard simulating still basically functions the same. It's up to the applications receiving these simulated events to act upon the extended flag we set. So again I don't care either way myself.